CHANGELOG.md

```diff
@@ -3,6 +3,205 @@
 <!-- prettier-ignore-start -->
 
 
+## 0.8.0
+
+Since we released [0.7.0](https://github.com/astral-sh/uv/releases/tag/0.5.0) in April, we've
+accumulated various changes that improve correctness and user experience, but could break some
+workflows. This release contains those changes; many have been marked as breaking out of an
+abundance of caution. We expect most users to be able to upgrade without making changes.
+
+This release also includes the stabilization of a couple `uv python install` features, which have
+been available under preview since late last year.
+
+### Breaking changes
+
+- **Install Python executables into the bin ([#14626](https://github.com/astral-sh/uv/pull/14626))**
+
+  `uv python install` now installs a versioned Python executable (e.g., `python3.13`) into a
+  directory on the `PATH` (e.g., `~/.local/bin`) by default. This behavior has been available under
+  the `--preview` flag since [Oct 2024](https://github.com/astral-sh/uv/pull/8458).
+
+  To install unversioned executables, e.g., `python3` and `python`, use the `--default` flag. The
+  `--default` flag has also been in preview, but is not stabilized in this release.
+
+  You can opt-out of this behavior with `uv python install --no-bin` or `UV_PYTHON_INSTALL_BIN=0`.
+
```

Comment:
I would add something like

"Please note that these `python3.x` commands will run the base Python installation with no installed packages, and they will not respect your project configuration. To run the right version of Python from your current project with dependencies installed, continue to use `uv python run`. There is no way to install additional packages into this base Python installation. If you need to run Python with dependencies without detecting settings from your current directory, you can use something like `uv python run --no-config --with numpy`. See [our documentation on ...]\(...) for more details."